README.md

```diff
@@ -82,7 +82,7 @@ Depending on your OS, Concrete ML may be installed with Docker or with pip:
 |            macOS 11+ (Intel)            |         Yes         |       Yes        |
 | macOS 11+ (Apple Silicon: M1, M2, etc.) |     Coming soon     |       Yes        |
 
-Note: Concrete ML only supports Python `3.8`, `3.9` and `3.10`.
+Note: Concrete ML only supports Python `3.8`, `3.9`, `3.10` and `3.11`.
```

Comment:
I would remove the "only support". But as you prefer.

```suggestion
Supported Python versions: `3.8`, `3.9`, `3.10` and `3.11`.
```

Reply:
@yuxizama / @andrei-stoian-zama : do you want to keep the "only support" or not? It means in particular we don't support 3.12

Reply:
As a reader, this is quite clear as a sentence. 
"Supported Python versions: `3.8`, `3.9`, `3.10` and `3.11`."
It implies that it doesn't support any other versions than the listed ones.

Reply:
ok, I've updated the doc accordingly